src/js/providers/video-listener-mixin.js

```diff
@@ -105,6 +109,9 @@ const VideoListenerMixin = {
     },
 
     playing() {
+        if (this.seeking) {
+            return;
+        }
```

Comment:
Make this a guard around `STATE_PLAYING` so that we don't miss firing `PROVIDER_FIRST_FRAME`:

```js
playing() {
    if (!this.seeking) {
         this.setState(STATE_PLAYING);
    }
    this.trigger(PROVIDER_FIRST_FRAME);
},
```